browser/brave_stats_updater_browsertest.cc

```diff
@@ -70,8 +70,9 @@ class BraveStatsUpdaterBrowserTest : public InProcessBrowserTest {
 
   void SetBaseUpdateURLForTest() {
     std::unique_ptr<base::Environment> env(base::Environment::Create());
-    brave::BraveStatsUpdater::SetBaseUpdateURLForTest(
-        embedded_test_server()->GetURL("/1/usage/brave-core").spec());
+    env->SetVar("BRAVE_USAGE_SERVER",
+        embedded_test_server()->host_port_pair().ToString());
+    env->SetVar("BRAVE_USAGE_LOCAL", "1");  // use http for local testing
```

Comment:
if this env is only for tests then I would add a SetEndpointForTesting method instead of the command line

Reply:
also there should be some new tests here to cover the new functionality